Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs

```diff
@@ -14,16 +14,93 @@ static void Main(string[] args)
             Console.ReadKey(true);
         }
 
-        private static void Test(string args, int expected)
-        {
+        private static void Test(string args, int expected)
+        {
             var result = FindDigit(args).Equals(expected) ? "PASS" : "FAIL";
-            Console.WriteLine($"{args} : {result}");
+            Console.WriteLine($"{args} : {result}");
         }
 
-        public static int FindDigit(string equation)
-        {
-            // Add your code here.
-            throw new NotImplementedException();
+        public static int FindDigit(string equation)
+        {
+            // Author : Aditi Rupade.
+
+            //Splitting string a*b=c into a,b,c
+            string[] splitEquation = equation.Split(new Char[] { '*', '=' });
+
+            //Case1: num1 has ?
+
+            if (splitEquation[0].Contains('?'))
+            {
+                 int num2 = 0, res = 0;
+
+                //Converting String to Integer
+                Int32.TryParse(splitEquation[1], out num2);
+                Int32.TryParse(splitEquation[2], out res);
+
+               //Checking for factors of result(c)
+                if (res % num2 == 0)

```

Comment:
`Remainder operator %` does implicit division to check remainder. 
- res % num2 == 0
- int temp = res / num2;
both of the above lines are doing division.

See if you can rewrite the code that does division only once and still solves the problem/condition.